deploy/sre-1270-rebalancing-infra-nodes/README.md

```diff
@@ -0,0 +1,2 @@
+References:
+* [Jira Epic](https://issues.redhat.com/browse/SDE-1270)
```

Comment:
Could you remove this private link, and write a description instead?

Reply:
Thank you for review. Make sense. I will remove that and fix the following ones.